### DISCUSSION TITLE
Where the equivalent of Main.Androids.cs in avalonia ?

### DISCUSSION THREAD

**Discussion by CastelloBrancoTecnologia** [AvaloniaUI/Avalonia#12718] [Q&A]
I need to permit clear text trafic on emulator using avalonia on android

what is the equivalent of bellow code in avalonia ?

Main.Androids.cs

namespace SplitTheTankApp.Droid {
    [Application(
        Label = "@string/ApplicationName",
        Icon = "@mipmap/icon",
        LargeHeap = true,
        HardwareAccelerated = true,
        Theme = "@style/AppTheme"
**#if DEBUG
        , UsesCleartextTraffic = true
#endif**
    )]
    public class Application : Microsoft.UI.Xaml.NativeApplication {

**Comment by maxkatz6** [comment]
https://github.com/AvaloniaUI/avalonia-dotnet-templates/blob/master/templates/csharp/xplat/AvaloniaTest.Android/MainActivity.cs#L9

Or you can specify it inside of the manifest file https://developer.android.com/guide/topics/manifest/application-element
